Content.Server/Bible/BibleSystem.cs

```diff
@@ -69,7 +70,7 @@ private void OnAfterInteract(EntityUid uid, BibleComponent component, AfterInter
 
             if (!_invSystem.TryGetSlotEntity(args.Target.Value, "head", out var entityUid) && !_tagSystem.HasTag(args.Target.Value, "Familiar"))
             {
-                if (_random.Prob(component.FailChance))
+                if (_random.Prob(component.FailChance) || HasComp<DiseaseZombieComponent>(args.Target))
```

Comment:
Assuming I'm reading the code right, this should probably smite the zombie instead of just doing nothing, but if you don't want to do that in this PR, then this should be fine as long as you add a TODO.

If you do implement it, then it should be more general like a list of components or a tag for all undead.

Reply:
This was actually meant to always smite, idk why i wrote it as something else. Though, i'll probably just remove this for the time being because that feature deserves to go in a separate pr for the more general zombie fixes and stuff that I need to do.